definitions/marts/looker_studio_marts/ecf/ls_ecf_correct_cohort_checker.sqlx

```diff
@@ -0,0 +1,255 @@
+config {
+    type: "table",
+    bigquery: {
+        partitionBy: "",
+        clusterBy: ["participant_role", "participant_profile_id"]
+    },
+    description: "This mart combines elements of the ECF Golden Thread and relevant declarations (submitted, eligible, payable, paid) for both ECTs and Mentors to assess if the currently assigned cohort is in line with the suggested cohort based on the induction start date for ECTs and the earliest declaration date for Mentors. \n \

```

Comment:
ECF_Golden_thread already is the chain of declarations for a single participant joined into a single record. ECF_Declarations operates similarly but instead of joining the declarations into a chain it has one row per declaration. I think you could use ECF_declarations here without needing to combine two tables to get the same information.